macros/plotTimeSeries.py

```diff
@@ -0,0 +1,67 @@
+
+parser = OptionParser()
+parser.add_option("-t", "--templatefile", type="string", dest="tfilename", default="listOfScanDates_gemPlotter.txt",
+                  help="Specify Template Filename", metavar="tfilename")
+parser.add_option("--vt1bump", type="int", dest="vt1bump", default=0,
+                  help="Specify the value of vt1bump", metavar="vt1bump")
+parser.add_option("--dataprefix", type="string", dest="data_prefix", default="/gemdata/",
+                  help="Specify Data Prefix", metavar="data_prefix")
+parser.add_option("--scanType", type="string", dest="scanType", default="scurve",
+                  help="Specify Data Prefix", metavar="scanType")
+
+(options, args) = parser.parse_args()
+
+chambers = ["GEMINIm01L1",
+            "GEMINIm27L1",
+            "GEMINIm27L2",
+            "GEMINIm28L1",
+            "GEMINIm28L2",
+            "GEMINIm29L1",
+            "GEMINIm29L2",
+            "GEMINIm30L1",
+            "GEMINIm30L2"]
+
+vt1bump = 'vt1bump'+str(options.vt1bump)
+data_prefix = options.data_prefix
+scanType = '/'+options.scanType+'/'
+template = options.tfilename
+
+print "Options: vt1bump=%s, template=%s, data_prefix=%s, scanType=%s"%(vt1bump, template, data_prefix, scanType)
+
+def makeInputList(chamberName):
+  ofname = data_prefix+chamberName+scanType+'listOfScanDates_gemPlotter.txt'
+  with open(template,"rt") as fin:
+    with open(ofname, "wt") as fout:
+      for line in fin:
+        fout.write(line.replace("GEMINI", chamberName)) 
+
+def makePlots(chamberName, vt1bump):
+  call_command = 'sudo -u gempro -i gemPlotter.py --infilename=/gemdata/'+chamberName+'/scurve/listOfScanDates_gemPlotter.txt --anaType=scurveAna --branchName=threshold --make2D --alphaLabels -c -a'
+  os.system(call_command)
+  call_command = 'sudo -u gempro -i gemPlotter.py --infilename=/gemdata/'+chamberName+'/scurve/listOfScanDates_gemPlotter.txt --anaType=scurveAna --branchName=noise --make2D --alphaLabels -c -a --axisMax=25'
+  os.system(call_command)
+  call_command = 'sudo -u gempro -i gemPlotter.py --infilename=/gemdata/'+chamberName+'/scurve/listOfScanDates_gemPlotter.txt --anaType=scurveAna --branchName=mask --make2D --alphaLabels -c -a --axisMax=1'
+  os.system(call_command)
+  call_command = 'sudo -u gempro -i gemPlotter.py --infilename=/gemdata/'+chamberName+'/scurve/listOfScanDates_gemPlotter.txt --anaType=scurveAna --branchName=maskReason --make2D --alphaLabels -c -a --axisMax=32'
+  os.system(call_command)
+  call_command = 'sudo -u gempro -i gemPlotter.py --infilename=/gemdata/'+chamberName+'/scurve/listOfScanDates_gemPlotter.txt --anaType=scurveAna --branchName=vthr --alphaLabels -c -a'
+  os.system(call_command)
+  call_command = 'sudo -u gempro -i mkdir -p /nfshome0/gempro/elog/timeSeriesPlots/'+chamberName+'/'+vt1bump+'/'
+  os.system(call_command)
+  call_command = 'sudo -u gempro -i mv /nfshome0/gempro/elog/summary*.png /nfshome0/gempro/elog/timeSeriesPlots/'+chamberName+'/'+vt1bump+'/'
```

Comment:
`/nfshome0/gempro` should **not** be hardcoded into this (`ELOG_PATH` is used to set this up usually)

Reply:
Do we have (or will we have in a near future) all these env variables everywhere?

Reply:
It makes more sense to put every new thing in with this in mind, and eventually ensure that nothing already in there has these paths hard-coded
So let's just assume that these variables will be ubiquitous

Reply:
Which would also mean taking the `sudo -u gempro -i` system calls and making them universal as well (without this what happens if `plotTimeSeries` is called as `gempro`?)